scripts/scil_volume_distance_map.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Compute distance map between two binary masks using KDTree. The distance map
+is the Euclidean distance from each voxel of the first mask to the closest
+voxel of the second mask.
+It's worst case scenario is two very big masks that are far appart or a very
+small mask that is completely inside a very big mask.
+
+Take this command as an example:
+  scil_volume_distance_map.py AF_L.nii.gz brain_mask.nii.gz \
+    AF_L_to_brain_mask.nii.gz
+
+We have a bundle and brain mask, the first is 100% inside the second.
```

Comment:
The example should feature "positive" examples instead of edge cases.

Reply:
I swap the order to start with the normal example, but I left the edge case since it is an important concept for distance map.